easybuild/framework/easyblock.py

```diff
@@ -2899,6 +2899,22 @@ def build_and_install_one(ecdict, init_env):
                 _log.info("Dumped fully processed easyconfig to %s", reprod_spec)
             except NotImplementedError as err:
                 _log.warn("Unable to dumped fully processed easyconfig to %s: %s", reprod_spec, err)
+            # also archive the relevant easyblocks
+            reprod_spec = os.path.join(new_log_dir, 'reprod', 'easyblocks')
+            for easyblock_class in inspect.getmro(app_class):
+                easyblock_path = inspect.getsourcefile(easyblock_class)
+                easyblock_splitpath = os.path.split(easyblock_path)
+                easyblock_basedir = easyblock_splitpath[0]
+                easyblock_filename = easyblock_splitpath[1]
+                framework_class_dir = os.path.join('easybuild', 'framework')
+                if easyblock_basedir.endswith(framework_class_dir):
+                    break
```

Comment:
@ocaisa This is basically where you hit `easyblock.py`? Maybe add a comment to clarify...